<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetToolLine</class>
 <widget class="QDialog" name="DialogSetToolLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Tool(Sub) Line</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>NAME : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_name_line">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_name_change_line">
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>370</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Object</string>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Feature</string>
    </property>
    <widget class="QSlider" name="horizontalSlider_feature_level_line">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>40</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Level :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_feature_level_line">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_feature_level_base_line">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_select_object_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Object</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_object_line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Object</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI Margin :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_detect_roi_margin_line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>pixel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>610</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Detect</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_detect_margin_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Margin :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_detect_threshold_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.35</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_margin_set_line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_margin_get_line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_threshold_get_line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_threshold_set_line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_get_base_image">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Base Image</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>150</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Set Area</string>
   </property>
   <widget class="QPushButton" name="pushButton_set_detect_area_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Detect Area</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>89</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Mask Area</string>
    </property>
    <widget class="QPushButton" name="pushButton_mask_push_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Push</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mask_pop_line">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Pop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mask_clear_line">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_mask_enable_inside_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>enable inside</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_detect_area_line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Detect Area</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>ID : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_id_line">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Zoom</string>
   </property>
   <widget class="QPushButton" name="pushButton_zoom_set_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zoom_reset_line">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_image_bg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_image_bg</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_7</zorder>
  <zorder>pushButton_get_base_image</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_image</zorder>
  <zorder>layoutWidget_line</zorder>
  <zorder>groupBox_8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
